modules/tinymce/src/core/test/ts/browser/content/EditorContentTest.ts

```diff
@@ -135,7 +138,7 @@ describe('browser.tinymce.core.content.EditorContentTest', () => {
     const editor = hook.editor();
     editor.setContent('<p>html</p>');
     editor.setContent('<p>new html</p>');
-    TinyAssertions.assertContent(editor, '<p>new html</p>');
+    assert.deepEqual(editor.getContent(), '<p>new html</p>');
```

Comment:
The wrong thing appears to have changed here... `TinyAssertions` is the correct thing to use here. It was the `Assertions` module above that was the issue. Also this doesn't need to be `deepEqual` it can just be `assert.equal`.

In saying that though, that part above could actually be this instead:
```
TinyAssertions.assertContent(editor, expected, { format: 'text' });
```

Reply:
Derp. Changed the ( hopefully ) correct one this time. And moved this one back to the original one.
No idea what I was thinking.